common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -138,7 +139,7 @@ public void onPageLoaded() {
 
                         // Inject the javascript string from testing. This should only be evaluated if we haven't sent
                         // an auth result already.
-                        if (mAuthResultSent == false && !StringExtensions.isNullOrBlank(mPostPageLoadedJavascript)) {
+                        if (!mAuthResultSent && !StringExtensions.isNullOrBlank(mPostPageLoadedJavascript)) {
                             if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
                                 mWebView.evaluateJavascript(mPostPageLoadedJavascript, null);
```

Comment:
@jbzdarkid Hi, I keep your evaluateJavascript here. It should not cause test unstable. I'll try more after this PR merged.

Reply:
<3 Thank you, I'm happy to help you with investigation if you need.